src/main/java/org/spburegistry/backend/dto/ProjectRequest.java

```diff
@@ -0,0 +1,41 @@
+package org.spburegistry.backend.dto;
```

Comment:
I think it is better to rename this class to ProjectRequestTO

Reply:
Done